Packages/Sites/Neos.NeosIo/Resources/Private/Translations/en/Main.xlf

```diff
@@ -17,6 +17,9 @@
             <trans-unit id="fundingBadges.filterType.onetime" xml:space="preserve">
                 <source>One Time</source>
             </trans-unit>
+            <trans-unit id="fundingBadges.filterType.abo" xml:space="preserve">
```

Comment:
"abo" makes me cinch, should be "subscription" in an english-language-context. But whatever, fix itself seems obvious.